package_control/package_manager.py

```diff
@@ -1265,7 +1265,7 @@ def install_package(self, package_name, is_dependency=False):
                     )
                     return None
                 else:
-                    os.rmdir(unpacked_package_dir)
+                    delete_directory(unpacked_package_dir)
```

Comment:
This seems weird to me. The code is only run if `clear_directory` succeeded, but if it was a symlink it will have been removed already. That means various os functions will error on the path. Might make sense to change the semantics of `clear_directory` to also remove the folder if it is empty. At least in the two cases indicated in this diff this would work out without issues.

Reply:
Yes, I'm not happy with semantics the more I look at the change.
That means that the changes would have to be more invasive...